README.md

```diff
@@ -47,7 +47,9 @@ The following example collects the traffic for an app in the Android emulator. I
 
```

Comment:
I feel like a major feature like managed emulators deserves an actual mention in the README, doesn't it? :D